Subject: [PATCH] loc_core: link liblog

libloc_core calls __android_log_print (through the LOC_LOG* macros) without
listing liblog. Up to Oreo libcutils brought it in; on Pie the link fails with
"undefined reference to '__android_log_print'". The other three modules already
list it.
---
--- a/core/Android.mk
+++ b/core/Android.mk
@@ -18,6 +18,7 @@
 LOCAL_SHARED_LIBRARIES := \
     libutils \
     libcutils \
+    liblog \
     libgps.utils \
     libdl
 
